deployment-config/chains/288.json

```diff
@@ -1,6 +1,4 @@
 {
```

Comment:
Let's add a "name" field that's unused but lets us know which actual chain these are